shared/middleware/shared_resources.rb

```diff
@@ -64,7 +64,7 @@ def cache(type)
   end
 
   # JavaScripts
-  get "/shared/js/*" do |path|
+  get "/shared/js/*" do |_path|
```

Comment:
Is `path` intentionally being shadowed? Not being familiar, I don't know if my fix is correct or masking a bug.

Reply:
Good question.  I suspect your fix is correct.